daemon/src/maker.rs

```diff
@@ -180,12 +180,13 @@ async fn main() -> Result<()> {
                                 tokio::spawn(in_taker_actor);
                                 tokio::spawn(out_msg_actor);
 
-                                maker_inc_connections_address.do_send_async(
-                                    maker_inc_connections_actor::NewTakerOnline {
+                                maker_inc_connections_address
+                                    .do_send_async(maker_inc_connections_actor::NewTakerOnline {
                                         taker_id,
                                         out_msg_actor_inbox,
-                                    },
-                                );
+                                    })
+                                    .await
+                                    .unwrap();
```

Comment:
Why is this not flagged as a "future must be used" by clippy?

Reply:
nobody knows. we should probably report that. I think it should be reported...

Reply:
I'll report it tomorrow. it took us a while to find that problem ;/